testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/SuiteRun.java

```diff
@@ -124,6 +126,9 @@ public Integer call()
         @Option(names = "--cli-executable", paramLabel = "<jar>", description = "Path to CLI executable " + DEFAULT_VALUE, defaultValue = "${cli.bin}")
         public File cliJar;
 
+        @Option(names = "--impacted-features", paramLabel = "<txt>", description = "Skip tests not using these features " + DEFAULT_VALUE, split = ",")
```

Comment:
`Skip tests not using these features`, Please phrase it without `not`, Maybe `Run tests tests with these features`?

Reply:
Done!